README.md

```diff
@@ -100,9 +102,19 @@ $ curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operato
 If you want to make changes to the HCO, here's how you can test your changes
 through [OLM](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md#installing-olm).
 
+The make target `cluster-sync-olm` automates the steps in this section:
+```shell
+$ make cluster-sync-olm
+```
+
+If you would rather want to understand the details, you can find it below.
+
+If you have made changes to the Operator and/or the webhook part of the code, first run `make build-operator` and/or 
+`make build-webhook`.
+
 Build the HCO container using the Makefile recipes `make container-build` and
 `make container-push` with vars `IMAGE_REGISTRY`, `REGISTRY_NAMESPACE`, and `CONTAINER_TAG`
-to direct it's location.
+to direct its location.
 
 To use the HCO's container, we'll use a registry image to serve metadata to OLM.
```

Comment:
since this can be used with `make cluster-sync-olm` I suggest moving this section up

Reply:
just before `make cluster-sync-olm`